src/pages/UserHome.jsx

```diff
@@ -35,6 +38,18 @@ const useStyles = makeStyles((theme) => ({
   addScholarship: {
     marginTop: theme.spacing(6),
   },
+  addTextAndLink: {
+    textAlign: 'center',
+  },
+  addIconSpacing: {
```

Comment:
`inboxIcon` is a better name. `inboxIcon` is more descriptive of what's being styled than `addIcon`. Also, not having `spacing` in name allows it to be used for styling the icon in non-spacing related ways.